showmore_dyn/showmore_dyn.js

```diff
@@ -0,0 +1,81 @@
+$(document).ready(function(){
+        handleNewWallItemBodies();
+
+        var mutationObserver = new MutationObserver(function(mutations) {
+                handleNewWallItemBodies();
+        });
+        mutationObserver.observe($("#content")[0], { attributes: false, characterData: false, childList: true, subtree: true, attributeOldValue: false, characterDataOldValue: false });
+});
+
+function handleNewWallItemBodies() {
+        $('.wall-item-body:not(.showmore-done)').each(function(i, el) {
+                $(el).addClass('showmore-done');
```

Comment:
As a general rule, if you're going to use `$(whatever)` multiple times, it's better to store its result in a variable. I like to prefix jQuery object variable with `$` to know they are active, like this:
```javascript
var $wall_item_body = $(this);
$wall_item_body.addClass(...);
```